service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/Configuration.java

```diff
@@ -297,7 +292,9 @@ public void setForceUpdate(Boolean forceUpdate) throws IllegalArgumentException
             throw new IllegalArgumentException("forceUpdate cannot be null");
         }
 
-        this.forceUpdate = forceUpdate;
+        /**
+         * Flip-flop helper for sending a forced update
+         */
```

Comment:
this comment is unattached to any field :D
it looks like unused field "forceUpdate" has been removed, this comment should be removed as well

Reply:
also, this setter is no longer setting any field, that doesn't look right

Reply:
pretty sure removed the comment but not sure why it showed up after merge.
Wil lremove it